### DISCUSSION TITLE
Audio Detection

### DISCUSSION THREAD

**Discussion by starsoccer** [8ff/firescrew#6] [Ideas]
Are there any plans to add other types of detection besides video. For instance audio detection for detecting a car horn, or TTS to convert a streams video to text, etc..

**Comment by chiefMarlin** [comment]
There isnt anything related to that on the roadmap at the moment but i am open to suggestions.
I am moving this to discussions, please provide some examples of things you would like to have implemented and how you would benefit from them

**Reply by starsoccer** [reply]
Sure so offhand a few use cases:

1. Animal Noises - Rather then capture animals from video, animals could be tagged by the sounds they make
2. Speaking/talking + TTS - This is useful for knowing when someone may be in a room but not visible on a camera. Its also useful for getting a transcript of a conversation. Also useful if someone is talking when there is no one present. For instance in a dark room
3. Fire Alarms/CO monoxide alarm - Alerting when alarms go off
4. Unexpected noise - For Notifications when sound or DB level is above a certain value
